teammapper-frontend/src/app/modules/start/map-list.component.html

```diff
@@ -9,8 +9,12 @@ <h4>Letzte Mindmaps</h4>
       }}</a>
     </span>
     <span matLine>
-      wird gelöscht am
-      {{ cachedMapEntry.cachedAdminMapValue.ttl | date : 'dd.MM.YYYY' }}</span
+      vom
```

Comment:
we need a fallback here as some old maps might not yet have the createdAt date. So best would be to check if this is available and otherwise do not show it or use the fallback of ttl

Reply:
ok sorry, update: if created at is not available, do not show this map on this page. so only maps where a createdAt value exists.

Reply:
Updated so it only shows maps where createdAt is set.